README.md

```diff
@@ -1,28 +1,78 @@
 # github-action-detect-unused-secrets
 
-GitHub Action that automatically detects unused secrets comparing the workflow secret names with the repo secret names.
+GitHub Action that detects unused secrets.
+Compares the secret names in your repository with the secret names under `.github/workflows`
 
 ## Example usage
 
-```yaml
-name: detect-unused-secrets
+The example below runs the action every Monday at 10am.
+Generates a token with required permissions which `secrets.GITHUB_TOKEN` does not have.
```

Comment:
no need to clarify this, or if you want to clarify it also clarify that secrets.GITHUB_TOKEN is the built-in token and that there's no way to give that token the necessary permissions